components/dashboard/challenge-card.tsx

```
@@ -1,10 +1,11 @@
-import { Calendar, Trophy, type LucideIcon } from "lucide-react";
-import { cn } from "@/lib/utils";
+import { Calendar, Trophy, Code, Users, Target } from "lucide-react";
+
+const cn = (...classes) => classes.filter(Boolean).join(' ');
 
 interface ChallengeCardProps {
 	title: string;
 	description: string;
-	icon: LucideIcon;
+	icon: React.ComponentType<any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)